BasicCardGame.js

```diff
@@ -1,81 +1,116 @@
-var deck = [];                          //Nachziehstapel
-var stack = [];                         //Ablagestapel
-var graveyard = [];                     //Friedhof für bereits ausgewertete Karten
-var enemyHealthPoints = [];             //NPC Lebenspunkte
-var playerHealthPoints = [];            //Spieler Lebenspunkte
-var numberOfHandCards;                  //Anzahl der maximalen Handkarten
-var enemyHandCards = [];                //Handkarten des NPC's
-var playerHandCards = [];               //Handkarten des Spielers
-
-
-window.onload = preGameFunction();      //automatischer Scriptstart bei aufrufen der Seite
-
-
-function preGameFunction() {            //vorbereitende Funktion
-
-    enemyHealthPoints, playerHealthPoints = [20,20];        //Festsetzen der Spielerlebenspunkte
-    numberOfHandCards = 5;                                  //Festsetzen der maximale Handkartenzahl
-    deck = ["Funken","Funken","Funken","Funken","Funken","Glut","Glut","Glut","Glut","Feuerbal","Feuerbal","Feuerbal","Sturm","Sturm","Sturm","Sturm","Sturm","Orkan","Orkan","Orkan","Panic Button","Panic Button","Panic Button","Panic Button","Blithschlag","Blithschlag","Blithschlag","Blithschlag","Erdbeben","Erdbeben","Erdbeben","Heilender Segen","Heilender Segen","Heilender Segen","Heilender Segen","Sintflut","Sintflut","Eiszeit","Eiszeit","Eiszeit"];
-   //Deck deklarieren
-    evaluationRound();                                     //Übergang in die Auswertungsrunde um Starthände zu verteilen
+var deck = []; //Nachziehstapel
+var stack = []; //Ablagestapel
+var graveyard = []; //Friedhof für bereits ausgewertete Karten
+var enemyHealthPoints = []; //NPC Lebenspunkte
+var playerHealthPoints = []; //Spieler Lebenspunkte
+var numberOfHandCards; //Anzahl der maximalen Handkarten
+var enemyHandCards = []; //Handkarten des NPC's
+var playerHandCards = []; //Handkarten des Spielers
+
+
+window.onload = preGameFunction(); //automatischer Scriptstart bei aufrufen der Seite
+
+
+function preGameFunction() { //vorbereitende Funktion
+
+  enemyHealthPoints,
+  playerHealthPoints = [20, 20]; //Festsetzen der Spielerlebenspunkte
+  numberOfHandCards = 5; //Festsetzen der maximale Handkartenzahl
+  deck = [
```

Comment:
Ich habe das Array für die Karten mal untereinander geschrieben, dann ist das übersichtlicher als in einer Zeile